main.c

```diff
@@ -207,7 +256,12 @@ static void create_output(struct slurp_state *state,
 	output->scale = 1;
 	wl_list_insert(&state->outputs, &output->link);
 
-	wl_output_add_listener(wl_output, &output_listener, output);
+	if (state->xdg_output_manager) {
```

Comment:
This won't do, because the xdg-output-manager global can be advertised after all outputs. We need to create xdg-outputs and add listeners after the first roundtrip in `main` (and roundtrip a second time to retrieve xdg-output properties).

Reply:
This check is still needed for hotplugged outputs though.

Reply:
Hmm, I don't think we support these at all. because we don't create layer-shell surfaces for those. So this would be a follow-up issue.

Reply:
Follow-up issue: https://github.com/emersion/slurp/issues/32